lib/web/mage/adminhtml/browser.js

```diff
@@ -102,6 +110,69 @@ define([
 
         },
 
+        /**
+         * Opens dialog for new media gallery
+         *
+         * @param {*} url
+         * @param {*} width
+         * @param {*} height
+         * @param {*} title
+         * @param {Object} options
+         */
+        openDialogNewMediaGallery: function (url, width, height, title, options) {
```

Comment:
Use "new" in  the name is bad practice. Please use words that describe behavior

Reply:
ok

Reply:
Can I ask you to change other places that have "new", like `is_new media_gallery`?